pkg/gps/GpsConsumer.php

```diff
@@ -110,7 +110,7 @@ private function getSubscription(): Subscription
 
     private function convertMessage(GoogleMessage $message): GpsMessage
     {
-        $gpsMessage = GpsMessage::jsonUnserialize($message->data());
```

Comment:
Looks like a BC break. Is it? Could you think of a backward compatible solution ?

Reply:
thank you for checking it out :) I have added the support for backward compatibility by default.